.github/workflows/ci.yml

```diff
@@ -6,6 +6,8 @@ jobs:
     runs-on: ubuntu-latest
     if: "!contains(github.event.head_commit.message, 'ci skip') && !contains(github.event.head_commit.message, 'skip ci')"
     name: Run unit tests
+    env:
+      NX_BRANCH: ${{ github.event.number || github.ref_name }}
```

Comment:
what are we aiming at here? Branch name for different triggers?

Reply:
This env var needs to be set so the nx cloud dashboard has a reference to the github branch. Apparently (I found this on their slack workspace) this is the way to do it. 🤔

Reply:
I mean is `github.event.number` empty in some cases so that we want to fallback to `github.ref_name`? In which cases is that?

Reply:
> I mean is github.event.number empty in some cases
Yeah, this workflow runs on push so I believe it's empty when doing anything that isn't associated with a PR (merging to main, tagging). In those cases we grab the reference to the branch/tag instead.